packages/lit-dev-cloudbuild-cleanup/src/cleanup.ts

```diff
@@ -0,0 +1,272 @@
+import {google} from 'googleapis';
+import {Octokit} from 'octokit';
+import {request} from 'gaxios';
+
+const ONE_WEEK_IN_MS = 7 * 24 * 60 * 60 * 1000;
+const ONE_WEEK_AGO = new Date(Date.now() - ONE_WEEK_IN_MS);
+
+const {PROJECT_ID, REPO_NAME, _DEPLOY_REGION} = process.env;
+
+if ([PROJECT_ID, REPO_NAME, _DEPLOY_REGION].some((env) => env === undefined)) {
+  console.log({PROJECT_ID, REPO_NAME, _DEPLOY_REGION});
+  throw new Error(
+    'missing one or more required environment variables: PROJECT_ID, REPO_NAME, _DEPLOY_REGION'
+  );
+}
+
+const sleep = (time: number) =>
+  new Promise((resolve) => setTimeout(resolve, time));
+
+async function main() {
```

Comment:
Could use top-level await here (Node 14.8+) so the `main` isn't required.

Reply:
I did that initially but the `for await` here can't be done top-level
https://github.com/lit/lit.dev/blob/179b9e2f636c98de076faf03573de4bb013a70bd/packages/lit-dev-cloudbuild-cleanup/src/cleanup.ts#L33

Reply:
Oh, weird, didn't know that wasn't supported.